mobile/repo.go

```diff
@@ -51,13 +50,12 @@ func checkWritable(dir string) error {
 	}
 
 	if os.IsPermission(err) {
-		return fmt.Errorf("cannot write to %s, incorrect permissions", err)
+		return errors.New(fmt.Sprintf("cannot write to %s, incorrect permissions", err))
```

Comment:
Sorry I might have caused confusion in my earlier comments:

When an error message does not require formatting, i.e. it's just a fixed string, then use `errors.New`. But when it requires formatting, like the case above, then use `fmt.Errorf`. So I recommend only changing uses of `fmt.Errorf` to `errors.New` when the error returned just has a fixed sting.

Reply:
oh you're right, I didn't realize the `fmt.Errorf` func returns an error type, that's why I used `fmt.Sprintf` with `errors.New`.
corrected it